README.md

```diff
@@ -96,12 +96,22 @@ $ npm install
 
 ### Browser support
 
-| Operating systems/Devices | Browser | Viewport |
-| --- | --- | --- |
-| Windows | ?? | ?? |
-| Mac OS X | ?? | ?? |
-| Phone 1 (iOS x.xx) | ?? | ?? |
-| Tablet 1 (iOS (x.xx) | ?? | ?? |
+| Operating systems/Devices | Browser | Priority | Breakpoints |
+| --- | --- | --- | --- |
+| Windows 7 | IE11 | **high** | all |
+| Windows 8 | IE11 | **high** | all |
+| Windows 10 | IE11 | **high** | all |
+| Windows 10 | Edge (newest) | **high** | all |
+| Windows 10 | Firefox (newest) | _medium_ | all |
+| Windows 10 | Chrome (newest) | _medium_ | all |
+| Mac OS X 10.13 | Safari | _medium_ | all |
+| Mac OS X 10.13 | Firefox and Chrome not required according to client | low | none |
```

Comment:
According to confluence doc they are required (low prio though)

Reply:
Done